prism_compile.c

```diff
@@ -1500,14 +1502,42 @@ pm_compile_defined_expr0(rb_iseq_t *iseq, const pm_node_t *node, LINK_ANCHOR *co
           pm_compile_defined_expr0(iseq, constant_path_node->parent, ret, src, popped, scope_node, dummy_line_node, lineno, true, lfinish);
           ADD_INSNL(ret, &dummy_line_node, branchunless, lfinish[1]);
           PM_COMPILE(constant_path_node->parent);
-        } else {
+        }
+        else {
           ADD_INSN1(ret, &dummy_line_node, putobject, rb_cObject);
         }
         ADD_INSN3(ret, &dummy_line_node, defined, INT2FIX(DEFINED_CONST_FROM),
                   ID2SYM(pm_constant_id_lookup(scope_node, ((pm_constant_read_node_t *)constant_path_node->child)->name)), PUSH_VAL(DEFINED_CONST));
         return;
       }
 
+      case PM_CALL_NODE: {
+        pm_call_node_t *call_node = ((pm_call_node_t *)node);
+        if (call_node->receiver) {
+          if (PM_NODE_TYPE_P(call_node->receiver, PM_SELF_NODE)) {
```

Comment:
Total nit, but the indentation is off here (should be 4 spaces).

Reply:
I think the style so far has been:

2 spaces between switch and case label, 2 spaces between case label and block, 4 spaces otherwise. Hopefully it's been consistent in that?

Reply:
@kddnewton no, it hasn't been consistent.  I _think_ my link above shows an example of the inconsistency.  AFAIK, this whole function is not consistent with the rest of the file.

Reply:
Ahh okay, ignore this then